HACKING.adoc

```diff
@@ -38,18 +38,21 @@ make runtop
 make tests
 ----
 
-6. Install in a new opam switch to try things out:
+6. Install in a new opam switch to try things out. With `opam` v2, create a local
+opam switch with the compiler installed from the current source directory:
 +
 ----
-opam compiler-conf install
+opam switch create . --empty
+opam install .
 ----
 +
-With opam 2, create a local opam switch with the compiler installed from
-the current source directory:
+If you would like others to use your changes push them to a public repo. They
+can then be installed with:
 +
 ----
-opam switch create . --empty
-opam install .
+opam switch create ocaml-my-modification --empty
+# Replace $VERSION by the trunk version and $REPO by your repo location.
+opam pin add ocaml-variants.$VERSION+my-modification git+https://$REPO#my-modification
```

Comment:
I understand the usefulness of the commands above, but I find it a bit confusing to have them here. The rest of the commands in this tutorial section are meant to be used from a directory containing a clone of the ocaml repository, and act on this respository. If I understand correct, the present commands can be run from anywhere and create a global (not local) switch to test a not-checked-out-yet compiler variant.

I was going to suggest that you move this section below, in the more general "Tips and Tricks"-style section, so that it gets off the hot path. In fact you have written similar information below. (```Testing with `opam` ```). So I would suggest to remove this paragraph+example here and keep only the section below.

Reply:
Agreed done.